src/Frontend/Page/Idea.hs

```diff
@@ -442,7 +451,8 @@ instance FormPage CreatorStatement where
     redirectOf (CreatorStatement idea) _ = U.viewIdea idea
 
     makeForm (CreatorStatement idea) =
-        "statement-text" .: (Markdown <$> DF.text (unMarkdown <$> creatorStatementOfIdea idea))
+        -- TODO: Translate
```

Comment:
"Statement des Autors" (I suspect that will be changed by QA, but I can't think of anything better.)